packages/pdf/src/RPA/PDF/utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import platform
+import time
+
+
+class Buffer:
```

Comment:
Is there a reason this was copied over from Desktop? I don't see it used anywhere